.circleci/config.yml

```diff
@@ -30,6 +30,10 @@ jobs:
           name: Test
           command: yarn run test:ci
 
+      - run:
+          name: yarn audit
+          command: yarn audit
```

Comment:
Fun fact: if you don't put `name` here, it defaults to the value of `command`.

Reply:
trying that now.